packages/compiler-core/src/errors.ts

```diff
@@ -90,7 +90,7 @@ export enum ErrorCodes {
   X_V_MODEL_ON_PROPS,
   X_INVALID_EXPRESSION,
   X_KEEP_ALIVE_INVALID_CHILDREN,
-
+  X_DEFINE_MODEL_NO_VARIABLE,
```

Comment:
```suggestion
  X_DEFINE_MODEL_NO_ASSIGNMENT,
```

I think this is more semantically clear.

Reply:
That's interesting. I will update it soon.